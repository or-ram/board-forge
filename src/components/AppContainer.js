import React, { Component } from 'react';
import Board from './Board';

export default class AppContainer extends Component {
  render () {
    return (
      <div id="app">
        <Board />
      </div>
    );
  }
}
