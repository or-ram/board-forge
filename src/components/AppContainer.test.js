import React from 'react';
import AppContainer from './AppContainer';

describe('AppContainer Test', () => {
  test('Correctly rendered', () => {
    const wrapper = shallow(
      <AppContainer />,
    );

    expect(wrapper).toMatchSnapshot();
  });
});
