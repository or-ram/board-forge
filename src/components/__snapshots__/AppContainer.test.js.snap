// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer Test Correctly rendered 1`] = `
<div
  id="app"
>
  <Board />
</div>
`;
